src/ProjectNeon/Assets/Enemies/EncounterBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class EncounterBuilder : MonoBehaviour
+{
+    [SerializeField]
+    private List<Enemy> possible;
+
+    [SerializeField]
+    private int difficulty;
+
+    public List<Enemy> generate()
+    {
+        /**
+         * @todo #52:30min Implement encounter generation after the encounter algorithm is defined, and then
+         *  return correct list of enemies in generate() method.
+         */
+        /**
+         * @todo #52:15min After #29 merge, wire encounter builder restult with BattleSetup so we can use the generated
+         *  enemies into the battle scene. BattleEnemiesSpawned  event must be triggered after they are added to
+         *  the scene.
+         */
+        if (difficulty < 0) difficulty = 1;
```

Comment:
This value changing is hidden from the designers. This violates the concept of "Don't Touch the Args".

Instead, you can limit the range of values in the inspector: https://www.reddit.com/r/Unity3D/comments/34mbrn/range_attribute_in_inspector_for_a_property/